packages/telemed-ehr/app/src/App.tsx

```diff
@@ -22,16 +22,10 @@ import { TelemedAdminPage } from './pages/TelemedAdminPage';
 import { useNavStore } from './state/nav.store';
 import EditInsurance from './telemed/features/telemed-admin/EditInsurance';
 import EditStatePage from './telemed/features/telemed-admin/EditState';
-import { isLocalOrDevOrTestingOrTrainingEnv } from './telemed/utils/env.helper';
 import { RoleType } from './types/types';
 import { AppointmentPage } from './pages/AppointmentPage';
 import AddSchedulePage from './pages/AddSchedulePage';
-
-const enablePhoton = false && isLocalOrDevOrTestingOrTrainingEnv;
-
-if (enablePhoton) {
-  import('@photonhealth/elements').catch(console.log);
-}
+import('@photonhealth/elements').catch(console.log);
```

Comment:
Why do we need this .catch()?

Reply:
it exists in PM, i'm not sure if i understand enough of TS to make that call, so will take your opinion on this